amazon/ionbenchmark/format_conversion_inaccurate.py

```diff
@@ -0,0 +1,42 @@
+import json
+import os
+
+import cbor_json
+
+from amazon.ion.json_encoder import IonToJSONEncoder
+from amazon.ion.simpleion import load
+
+# The file names you want to convert or be converted
+ion_file = 'test_ion'
+json_file_list = 'test_json'
+cbor_file_list = 'test_cbor'
+
+# Flags to control whether a file in a specific format needs to be generated.
+convert_json = True
+convert_cbor = True
+
+# Flag to control if the JSON file is delimited by a newline.
+generate_jsonl = True
+
+with open(ion_file, 'br') as fp:
```

Comment:
For posterity, can you reply here to explain what was wrong with this technique?

Reply:
Sure, the reasons are:
1. Since JSON doesn't support multiple top-level objects, we used to store all JSON object within a list. Now we iterate each top level values and write them one by one. 
2. CBOR conversion was wrong. We used jsonable bytes for JSON/CBOR conversion instead of Python objects that are converted from JSON objects so that the resulted CBOR data is incorrect.